### DISCUSSION TITLE
Send Confirm Email issues

### DISCUSSION THREAD

**Discussion by quocbang** [quocbang/data-flow-sync#9] [Ideas]
về phần gửi mail này ae có nghĩ là sẽ gửi otp về cho user là tốt nhất không?
tạo id và otp trên redis rồi user nhập otp confirm nếu trùng là được otp này sống khoảng 60s thôi sẽ bị xóa thì báo otp hết hạn hoặc không đúng

@leducthai 

cc @caosown

**Comment by leducthai** [comment]
@quocbang cũng được để tìm hiểu xem
